frontend/src/components/posts/comment/CommentView.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react/no-danger */
+import * as React from 'react'
+import Box from '@mui/material/Box'
+import Modal from '@mui/material/Modal'
+import LoadingButton from '@mui/lab/LoadingButton'
+import styles from './comment.module.scss'
+import SimpleUserDetails from '../../user/simpledetails/SimpleUserDetailsController'
+import PostInput from '../postinput/PostInputController'
+import processMentions from '../../../functions/mentions'
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 300,
+  bgcolor: 'background.paper',
+  boxShadow: 24,
+  p: 4,
+}
+
+export default function BasicModal({
+  postData,
+  setPostTags,
+  setPostHandles,
+  setPostText,
+  loading,
+  submitForm,
+  setNewTags,
+  tags,
+  handles,
+}) {
+  const [open, setOpen] = React.useState(true)
+
+  const handleClose = () => setOpen(false)
+
+  return (
+    <div>
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box sx={style}>
+          <SimpleUserDetails
+            condensed
+            user={postData.author}
+            time={postData.timestamp}
+          />
+          <div
+            dangerouslySetInnerHTML={{
+              __html: processMentions({
+                content: postData.content ?? '',
+                tags,
+                handles,
+                tagStyle: styles.tagStyle,
+                handleStyle: styles.handleStyle,
+              }),
+            }}
+            style={{ marginTop: 10, marginLeft: 70 }}
+          />
+          <div
+            dangerouslySetInnerHTML={{
+              __html: processMentions({
+                content: `Replying to @${postData.author.username} ` ?? '',
+                tags,
+                handles,
+                tagStyle: styles.tagStyle,
+                handleStyle: styles.handleStyle,
+              }),
+            }}
+            style={{ marginTop: 25 }}
+          />
```

Comment:
Can you style this using a 
```html
<p>Replying to <a href="/user/???"">@Author</a></p>
```

The processMention runs an intensive loop would be better if we don't use it for something we can hard code.

Reply:
Cool I've made those changes